chainer_pytorch_migration/ignite/__init__.py

```diff
@@ -1,2 +1,15 @@
+from packaging import version
```

Comment:
`packaging` is not a standard module. How about parsing manually?
I think it should be enough to parse up to the minor version and it's unlikely each component may contain non-digit characters.

Reply:
Ok, I am comparing now only the second component of the version string.